devVerifiers.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: subpage
+title: "Developer Docs for VC Verifiers"
+permalink: /developers/verifiers
+---
+## CHAPI for VC Verifiers
+CHAPI integrates easily into verifier websites, allowing your site to request Verifiable Credentials presentations from a user's digital wallet
+
+## Resources
+- **Example Code**: the [chapi-demo-verifier](https://github.com/credential-handler/chapi-demo-verifier) contains a full example implementation and is referenced throughout this guide. 
+- **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed functionality in the browser.
+
+## Import the CHAPI Polyfill into your Verfier Site
+If you're working in vanilla JavaScript, you can add the `navigator.credentials` and `credentialHandlerPolyfill` globals to your code and then load the polyfill library:
+
+```
+<script src="https://unpkg.com/credential-handler-polyfill@3.0.0/dist/credential-handler-polyfill.min.js"></script>
+
+<script>
+credentialHandlerPolyfill.loadOnce();
+</script>
+```
+<p class="code-annotation">
+  <a href="https://github.com/credential-handler/chapi-demo-verifier/blob/master/index.html"
+  target="_blank" rel="noopener noreferrer"> chapi-demo-verifier/index.html </a>
+</p>
+
+Or, if you're developing on Node.js, add the credential-handler-polyfill library to your project...
+
+```
+npm i credential-handler-polyfill@3.0.0
+``` 
+
+and then import and load the polyfill library as follows:
+
+```
+import * as CredentialHandlerPolyfill from 'credential-handler-polyfill';
+
+await CredentialHandlerPolyfill.loadOnce();
+console.log('Ready to work with credentials!');
+```
+## Construct a Web Credential Query
+A verifier can request credentials over CHAPI by forming a `VerifiablePresentationRequest` object, wrapped with additional information for working with *Web Credentials*.  The example below illustrates a `QueryByExample`, which is wrapped in a `web` object.  The `recommendedHandlerOrigins` parameter allows verfiers to suggest Credentail Handlers (e.g. digital wallets) for the user to present the requested data. 
+
+```
+const credentialQuery = {
+  "web": {
+    "VerifiablePresentation": {
+            "query": [{
+                "type": "QueryByExample",
+                "credentialQuery": {
+                    "reason": "Please present a UniversityDegreeCredential for JaneDoe.",
+                    "example": {
+                        "@context": [
+                            "https://w3id.org/credentials/v1",
+                            "https://www.w3.org/2018/credentials/examples/v1"
+                        ],
+                        "type": ["UniversityDegreeCredential"],
+                        "credentialSubject": {
+                            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+                        }
```

Comment:
```suggestion
```

Wondering if it would be better to simplify the example here.

Reply:
Yes, it would.  I'm planning to talk with Dave when it's convenient about this object, and then I can do a better job of extracting and explaining the relevant concepts (instead of just providing an example).  Thinking this'll be in the next round of revisions.